### PR TITLE
S005-T78 Added GET request for orders

### PR DESCRIPTION
**Pull request by lee32147**

### CODE REVIEW

**Comment by lee32147** [comment]
S005-T78 Added GET request for orders

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    private boolean active;
+    private Double amount;
+    private Double amountLess;
+    private Double amountMore;
+    private Date creationDateLess; //java.util.Date, możliwe do zmiany
+    private Date creationDateMore; //java.util.Date, możliwe do zmiany
+    private Date dateClosingLess; //java.util.Date, możliwe do zmiany
+    private Date dateClosingMore; //java.util.Date, możliwe do zmiany
+    private Date dateExpirationLess; //java.util.Date, możliwe do zmiany
+    private Date dateExpirationMore; //java.util.Date, możliwe do zmiany
```
Comment (Calharad) [inline benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java:22]: I think better will be [OffsetDateTime](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/OffsetDateTime.html) (It is from Java 8, while Date is from Java 1.1, so it is quite hard to use)

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderTO.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class OrderTO {
+    private Long amount;
+    private Date dateClosing; //java.util.Date, możliwe do zmiany
+    private Date dateCreation; //java.util.Date, możliwe do zmiany
+    private Date dateExpiration; //java.util.Date, możliwe do zmiany
```
Comment (Calharad) [inline benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderTO.java:15]: OffsetDateTime?

**Review by Calharad — COMMENTED** [review]
Added some comments
Please check my small fix added directly to branch S005-T75 (pull request [here](https://github.com/AI-2020-2021-LAB1/stock-exchange-app-benchmark/pull/14) for quick info), I forgot to add something to filters. They match to query parameters from [there](http://193.33.111.196:8000/swagger-ui.html?urls.primaryName=default#/Stocks/getStocksUsingGET) (I hope you'll understand what do they mean, if not, write there or on Messenger)

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+@Getter
+@Setter
+public class OrderFiltersTO {
+    private String abbreviation;
+    private boolean active;
+    private Double amount;
+    private Double amountLess;
+    private Double amountMore;
+    private Date creationDateLess; //java.util.Date, możliwe do zmiany
```
Comment (KrzysztofLesniak) [inline benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java:17]: The same problem, Date -> OffsetDateTime

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java (hunk 1 above)
Comment (pabloducato) [reply benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java]: It was resolved yesterday by @lee32147

File: benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderTO.java (hunk 2 above)
Comment (pabloducato) [reply benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderTO.java]: Resolved yesterday by @lee32147

**Review by pabloducato — APPROVED** [review]
Review ok

**Pull request merged**